src/api/auth/signup.ts

```diff
@@ -1,12 +1,10 @@
-import { DOMAIN_URL } from "@/common/constants";
+import { DOMAIN_URL, headers } from "@/common/constants";
 import { SubmitFormData } from "@/types/form";
 
-const headers = { "Content-Type": "application/json; charset=utf-8" };
-
 export async function signupUser(userData: SubmitFormData) {
   const res = await fetch(`${DOMAIN_URL}/auth/sign-up`, {
     method: "POST",
-    headers,
+    headers: headers,
```

Comment:
```suggestion
    headers,
```
간단한거지만 위와 같이 표현할 수도 있어요 !